scripts/kube-hunter-manifests/config.yaml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
We could consider using helm chart in the future: https://github.com/helm/charts/tree/master/stable/kube-hunter.

Reply:
Yep, but it lacks psp related settings.